src/screens/Settings/Experimental/FeatureRow.js

```diff
@@ -21,7 +23,7 @@ const FeatureRow = ({ feature }: Props) => {
   return (
     <SettingsRow
       event={`${feature.name}Row`}
-      title={feature.title}
+      title={type === "integer" ? `${feature.title}` : feature.title}
```

Comment:
mmh why is this needed?

Reply:
Isn't it the same thing?

Reply:
I wanted to add the current provider into brackets after the title `Provider (5)` but I forgot, maybe not essential so I will remove